invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/FundingField.js

```diff
@@ -108,46 +110,52 @@ function FundingFieldForm(props) {
             );
           })}
         </List>
-        <FundingModal
-          searchConfig={searchConfig}
-          trigger={
-            <Button
-              type="button"
-              key="custom"
-              icon
-              labelPosition="left"
-              className="mb-5"
-            >
-              <Icon name="add" />
-              {i18next.t("Add award")}
-            </Button>
-          }
-          onAwardChange={(selectedFunding) => {
-            formikArrayPush(selectedFunding);
-          }}
-          mode="standard"
-          action="add"
-          deserializeAward={deserializeAward}
-          deserializeFunder={deserializeFunder}
-          computeFundingContents={computeFundingContents}
-        />
-        <FundingModal
-          searchConfig={searchConfig}
-          trigger={
-            <Button type="button" key="custom" icon labelPosition="left">
-              <Icon name="add" />
-              {i18next.t("Add custom")}
-            </Button>
-          }
-          onAwardChange={(selectedFunding) => {
-            formikArrayPush(selectedFunding);
-          }}
-          mode="custom"
-          action="add"
-          deserializeAward={deserializeAward}
-          deserializeFunder={deserializeFunder}
-          computeFundingContents={computeFundingContents}
-        />
+
+        <Overridable id="InvenioVocabularies.FundingField.AddAward.Button">
```

Comment:
```suggestion
        <Overridable id="InvenioVocabularies.FundingField.FundingModal.container">
```

Reply:
We actually need these separate ids on individual elements since we just want to hide one of these buttons (we only want to allow adding custom awards in our project)

Reply:
I understand your point, and you are right on the fact we should allow overriding each of the elements as well.  In this case, however, the overridable ID should be added in the component itself - either on the `<Button>` in line 118 or in`FundingModal`. While we are adding overridable IDs as wrappers (parent of several "subcomponents") then we identify it as a "container" - otherwise we have

`InvenioVocabularies.FundingField.AddAward.Button`
`InvenioVocabularies.FundingField.AddCustom.Button`

both wrapping two instantiations of `FundingModal` component, which is semantically incorrect -  the naming does not mention the modal itself whatsoever. This naming convention we reserved to be associated with a component definition, not instantiation, to avoid misunderstandings and inconsistencies in the naming.

I am happy to explain it more in details on how this naming convention was planned to be used over a call.